aries-site/src/examples/templates/Card/Card.js

```diff
@@ -14,6 +14,18 @@ import {
 } from 'grommet';
 import { adjustPad } from './utils';
 
+//  demo purposes to make a skeleton icon for loading
+const IconSkeleton = () => (
+  <Box
```

Comment:
really this should instead either directly use `<Skeleton/>` or a `<Box background="whatever"/>` rather than specifying the skeleton prop (I know I'm trying to avoid specific uses of Skeleton). You should be able to avoid the conditional around where it's used if it's something the icon can be inside anyway.

Reply:
changed this let me know what you think with the `icon`